### PR TITLE
Docs: specify XML document name in xml.etree.elementtree example

### PR DESCRIPTION
**Pull request by moshekaplan**
The xpath-support section references that it uses the `countrydata` XML document located in the Parsing XML section.
However, the Parsing XML section does not explicitly state that the XML document is named `countrydata`. 
This change adds the name `countrydata` to the Parsing XML section.

### CODE REVIEW

**Comment by moshekaplan** [comment]
Can someone please add `skip issue` and `skip news` labels?

**Comment by moshekaplan** [comment]
Can someone please add `skip issue` and `skip news` labels?

File: Doc/library/xml.etree.elementtree.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ and its sub-elements are done on the :class:`Element` level.
 Parsing XML
 ^^^^^^^^^^^
 
-We'll be using the following XML document as the sample data for this section:
+We'll be using the following ``countrydata`` XML document as the sample data for this section:
```
Comment (erlend-aasland) [inline Doc/library/xml.etree.elementtree.rst]: How about this?

```suggestion
We'll be using the fictive :file:`country_data.xml` XML document as the sample data for this section:
```

Comment (moshekaplan) [reply Doc/library/xml.etree.elementtree.rst]: Sure, that'd be fine too. Do you want to add it or would you prefer if I did?

**Review by erlend-aasland — APPROVED** [review]
Thanks; sorry that it took so long before you got a review.

**Comment by moshekaplan** [comment]
> Thanks; sorry that it took so long before you got a review.

No worries, thanks for getting around to it 😄

**Pull request merged**